app/src/main/java/woowacourse/paint/DrawingCanvas.kt

```diff
@@ -0,0 +1,76 @@
+package woowacourse.paint
+
+import android.annotation.SuppressLint
+import android.content.Context
+import android.graphics.Canvas
+import android.graphics.Color
+import android.graphics.Paint
+import android.util.AttributeSet
+import android.view.MotionEvent
+import android.view.View
+import woowacourse.paint.model.DrawingElement
+import woowacourse.paint.model.DrawingHistory
+
+class DrawingCanvas @JvmOverloads constructor(
+    context: Context,
+    attrs: AttributeSet? = null,
+) : View(context, attrs) {
+
+    private val paletteHistory = DrawingHistory()
+    private var drawingElement = DrawingElement()
+
+    init {
+        isFocusable = true
+        isFocusableInTouchMode = true
+        initSetupPaint()
+    }
+
+    override fun onDraw(canvas: Canvas) {
+        super.onDraw(canvas)
+        paletteHistory.history.forEach {
```

Comment:
```suggestion
        paletteHistory.forEach {
```
paletteHistory의 캡슐화를 시도해보는건 어떤가요??